corehq/apps/app_manager/views/forms.py

```diff
@@ -809,6 +812,12 @@ def get_form_view_context(
         'session_endpoints_enabled': toggles.SESSION_ENDPOINTS.enabled(domain),
         'module_is_multi_select': module.is_multi_select(),
         'module_loads_registry_case': module_loads_registry_case(module),
+        'case_property_warning': {
+            'count': case_property_count,
+            'limit': _get_case_property_limit(domain),
+            'show': case_property_count > _get_case_property_limit(domain),
```

Comment:
I assume calling `_get_case_property_limit()` twice is not likely to cause a performance issue, but seems kinda wasteful since it's a DB hit (or two). Maybe worth storing the result in a variable to avoid the second call?